Sources/MapboxMapsSwiftUI/MapConfiguration.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+import MapboxMaps
+
+@available(iOS 13.0, *)
+struct MapConfiguration {
```

Comment:
nit: Lets organize the code here in a way, that you can see MapConfiguration and it's internal structs in one place, and all the extensions are left aside:

```swift
@available(iOS 13.0, *)
struct MapDeps {
    struct Actions {
        var onMapLoaded: MapLoadedAction?
        var onMapTapGesture: Map.TapAction?
        var tapActionsWithQuery = [Map.TapActionWithQueryPair]()
    }

    struct StyleURIs {
        let light: StyleURI
        let dark: StyleURI?
    }

    var actions = Actions()
    var styleURIs = StyleURIs(light: .streets, dark: nil)
    var cameraBounds = CameraBoundsOptions()
    var getstureOptions = GestureOptions()
}

extension MapDeps.StyleURIs {
    func effectiveURI(with colorScheme: ColorScheme) -> StyleURI { /*...*/ }
}
```